# Cartesian movement sequence message
Header header
time end
cartesian_move[] moves